Task 6/script.js

```diff
@@ -0,0 +1,41 @@
+/* ------------------------------ TASK 6 -----------------------------------
+Turimas "users" masyvas. 
+
+Parašykite funckijas, kurios atlikas nurodytas užduotis:
+1. funkcija "getUserAverageAge" - kaip argumentą priims masyvą ir duoto masyvo 
+atveju grąžins visų "users" amžiaus visurkį kaip skaičių.
+2. funkcija "getUsersNames" -  kaip argumentą priims masyvą ir duoto masyvo 
+atveju grąžins visų "users" vardus naujame masyve pvz., ['John Smith', 'Ann Smith'..].
+-------------------------------------------------------------------------- */
+
+const users = [
+  { id: "1", name: "John Smith", age: 20 },
+  { id: "2", name: "Ann Smith", age: 24 },
+  { id: "3", name: "Tom Jones", age: 31 },
+  { id: "4", name: "Rose Peterson", age: 17 },
+  { id: "5", name: "Alex John", age: 25 },
+  { id: "6", name: "Ronald Jones", age: 63 },
+  { id: "7", name: "Elton Smith", age: 16 },
+  { id: "8", name: "Simon Peterson", age: 30 },
+  { id: "9", name: "Daniel Cane", age: 51 }
+];
+
+/////   Task 1
+
+const getUserAverageAge = users.reduce((acc, curr, index, array) => {
+  const sum = array.reduce((total, value) => total + value.age, 0);
+
+  const avg = Math.round(((sum / array.length) * 100) / 100);
```

Comment:
Pertekliniai skliausteliai ties pirmu dalybos veiksmu. Kadangi dalyba ir po jos einanti daugyba yra vienodos reikšmės operacijos, viskas vyktų iš kairęs į dešinę pusę, kaip ir jūs norite.